app/services/malware_scanner.rb

```diff
@@ -0,0 +1,46 @@
+require 'open3'
+
+class MalwareScanner
+  ClamdscanError = Class.new(StandardError)
```

Comment:
```suggestion
  class ClamdscanError < StandardError; end
```

I feel that since no other time we do this in the editor we're doing it this way maybe we should do the above instead?